README.md

```diff
@@ -21,3 +21,40 @@ git checkout development
 
 Good luck, and have a great day!
 
+## Docker
+
+If you don't want to worry about installing our dependencies, 
+you can create a build of CAM-SIMA using docker. To do so, you will need to 
+download and install [docker desktop](https://docs.docker.com/desktop/).
+
+#### Windows
+Follow the installation instructions for Windows [here](https://docs.docker.com/desktop/install/windows-install/)
+
+
+#### Linux
+Follow the installation instructions for Linux [here](https://docs.docker.com/desktop/install/linux-install/)
+
+#### MacOS
+Follow the installation instructions for Mac [here](https://docs.docker.com/desktop/install/mac-install/)
+
+### Download docker
+
+### Build the docker file
+1. First, build the esmf docker image. You must tag the build with `esmf`
+```
+cd docker
+docker build -f Dockerfile.esmf -t esmf .
```

Comment:
On docker02.cgd.ucar.edu, I get the following error:


docker02$ docker build -f Dockerfile.esmf -t esmf .
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/build?buildargs=%7B%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile.esmf&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&shmsize=0&t=esmf&target=&ulimits=null&version=1": dial unix /var/run/docker.sock: connect: permission denied

Is this a problem on this side or the docker02 machine side?  I'm guessing the machine side, but is purely a guess on my part

Reply:
Hi @cacraigucar, that is definitely a machine-side issue.  I am guessing that you'll just need to be added to the `docker` group that @mwaxmonsky pointed out, but if that doesn't work then please let me know.

Reply:
I submitted the issue (with the proposed solution) to my help ticket last night right after I got the @mwaxmonsky email.  Hopefully they respond back soon (though today was the morning of updates for the IT staff)

Reply:
Now that I have the docker group, I get the following error messages (which I think are probably due to this setup).  (It took almost half an hour to do all this compilation)

make[8]: Leaving directory '/esmf-8.4.2/mod/modg/Linux.gfortran.64.mpich.default'
make[7]: *** [/esmf-8.4.2/build/common.mk:2283: esmflib] Error 2
make[7]: Leaving directory '/esmf-8.4.2/mod/modg/Linux.gfortran.64.mpich.default'
make[6]: *** [/esmf-8.4.2/build/common.mk:2279: tree_lib] Error 2
make[6]: Leaving directory '/esmf-8.4.2/src/Infrastructure/IO/src'
make[5]: *** [/esmf-8.4.2/build/common.mk:3771: tree] Error 1
make[5]: Leaving directory '/esmf-8.4.2/src/Infrastructure/IO'
make[4]: *** [/esmf-8.4.2/build/common.mk:3771: tree] Error 1
make[4]: Leaving directory '/esmf-8.4.2/src/Infrastructure'
make[3]: *** [/esmf-8.4.2/build/common.mk:3771: tree] Error 1
make[3]: Leaving directory '/esmf-8.4.2/src'
make[2]: *** [/esmf-8.4.2/build/common.mk:3771: tree] Error 1
make[2]: Leaving directory '/esmf-8.4.2'
make[1]: *** [/esmf-8.4.2/build/common.mk:2246: build_libs] Error 2
make[1]: Leaving directory '/esmf-8.4.2'
make: *** [/esmf-8.4.2/build/common.mk:2238: lib] Error 2
The command '/bin/sh -c wget -q https://github.com/esmf-org/esmf/archive/refs/tags/v${ESMF_TAG}.tar.gz &&     tar zxf v${ESMF_TAG}.tar.gz &&     cd esmf-${ESMF_TAG} &&     make info &&     make -j 8 &&     make install' returned a non-zero code: 2

Reply:
This error is because docker02 is not robust enough to build on.  Currently users with Windows laptops do not have access to a facility to build docker images.  I was able to test the non-build steps after @mwaxmonsky loaded the files onto docker02.